frontends/web/src/locales/en/app.json

```diff
@@ -311,7 +311,9 @@
       "noExchanges": "Sorry, there are no available exchanges in this region.",
       "region": "Region",
       "selectRegion": "Select region",
-      "title": "Buy {{name}}"
+      "title": "Buy {{name}}",
+      "bestDeal": "Best Deal",
+      "fast": "Fast"
```

Comment:
there is a linter check for translations, to fix just do: `make locize-fix` or alternatively (if you don't have locize installed globally) do: `npx locize-cli format frontends/web/src/locales --format json`